source/WorkingPractices/planning_your_change.rst

```diff
@@ -50,9 +50,9 @@ and things to avoid, so it is useful for the developer to be aware of these.
 * `PEP 8 (Python) <https://legacy.python.org/dev/peps/pep-0008/>`_
 
 **Who will SciTech review the change?** This is a useful consideration as not everyone who uses the
-repository has the knowledge or experience to review every ticket that is being developed. It can be worth
-identifying a SciTech reviewer before starting work to avoid having developed the ticket, only to
-find nobody is available to review it.
+repository has the knowledge or experience to review every ticket that is being developed. Get in
+touch with your SciTech reviewer early in the process as they will have valuable insights to positively
```

Comment:
Identify and get in touch with a suitable SciTech reviewer early in the process ...

Q - how do we prevent the SciTech being the Author by proxy ? Do we offer any advice on this ?

Reply:
Good question. I've added an "important" box to the top of the reviews page highlighting that the reviewer needs to remain impartial, hopefully this covers it - see what you think. 

All the extra emphasis on talking to people early comes in trying to incorporate the spirit of the old LFRic "design review" into the current process since its mostly there anyway and doesn't need to be another person when you already have code owners and reviewers look at your change. The trick is to talk to them before you go too far.... so just trying to stick a highlighter on that.